chain/network/src/stats/metrics.rs

```diff
@@ -348,19 +349,36 @@ pub(crate) static ALREADY_CONNECTED_ACCOUNT: Lazy<IntCounter> = Lazy::new(|| {
     .unwrap()
 });
 
-pub(crate) fn record_routed_msg_metrics(clock: &time::Clock, msg: &RoutedMessageV2) {
-    record_routed_msg_latency(clock, msg);
+pub(crate) fn record_routed_msg_metrics(
```

Comment:
add a comment (especially for the fastest field)

Reply:
done